lib/rubocop/cop/rails/action_controller_testing.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Using ActionController::TestCase is discouraged and should be replaced by
```

Comment:
Can you enclose the module name in backticks?

```suggestion
      # Using `ActionController::TestCase` is discouraged and should be replaced by
```